pkg/kubelet/kubelet_network.go

```diff
@@ -89,6 +90,120 @@ func (kl *Kubelet) providerRequiresNetworkingConfiguration() bool {
 	return supported
 }
 
+func omitDuplicates(kl *Kubelet, pod *v1.Pod, combinedSearch []string) []string {
+	uniqueDomains := map[string]bool{}
+
+	for _, dnsDomain := range combinedSearch {
```

Comment:
I've added unit test for checking return value of formDNSSearch, which consists of omitDuplicates call and formDNSSearchFitsLimits, you think it's not enough?